spec.html

```diff
@@ -14017,7 +14017,7 @@ <h1>Rules of Automatic Semicolon Insertion</h1>
             `throw` [no LineTerminator here] Expression[+In, ?Yield, ?Await] `;`
 
           ArrowFunction[In, Yield, Await] :
-            ArrowParameters[?Yield, ?Await] [no LineTerminator here] `=&gt;` ConciseBody[?In]
```

Comment:
We're inconsistent between using `=&gt;` and `=>`. Are there any `=&gt;`s left after this PR?

Reply:
I think we should always write `>` as `&gt;` when not part of an HTML tag.

Reply:
> Are there any `=&gt;`s left after this PR?

One, in `OtherPunctuator`, which uses `&gt;` consistently for all punctuators.

> I think we should always write `>` as &gt; when not part of an HTML tag.

Strong disagree. It's less readable for no benefit.

Reply:
Agree with @bakkot on not using `&gt;`. I prefer `=>`.